pack/ramp-community.yml

```diff
@@ -0,0 +1,25 @@
+display_name: RediSearch 2
+capability_name: Search and query
+author: RedisLabs
+email: meir@redislabs.com
+description: High performance search index on top of redis
+homepage: http://redisearch.io
+license:  Redis Source Available License 2.0 (RSALv2) or the Server Side Public License v1 (SSPLv1)
+command_line_args: ""
+compatible_redis_version: "7.4"
```

Comment:
Is `compatible_redis_version` relevant for `min_redis_pack_version` 7.6

Reply:
Yes. Included in PR: https://github.com/RediSearch/RediSearch/pull/4567